### PR TITLE
Add shortcut for next workspace needing attention

### LINKED ISSUE #2519: feat: jump to the next workspace needing attention
## Problem

When several agent sessions are running, reviewing completed work or responding to blocked sessions requires scanning the sidebar and selecting each workspace manually.

## Expected flow

Add a customizable keyboard shortcut that jumps to the next workspace that either needs input or is ready to review.

The navigation should:

- use the current cross-host workspace set
- prioritize the existing status order
- wrap after the final matching workspace
- do nothing when no workspace matches
- remain current while the sidebar is hidden

The default shortcut should be Option+R on macOS and Alt+R elsewhere.

### PR DESCRIPTION
**Pull request by colonelpanic8**
Closes #2519

## Problem

Moving through agent sessions that need input or are ready to review requires scanning the sidebar and selecting each workspace manually.

## Changes

- add a customizable Option+R shortcut on macOS (Alt+R elsewhere)
- navigate through `needs_input` and `attention` workspaces in the existing status order, with wraparound
- keep shortcut targets current across hosts and while the sidebar is hidden
- route through the existing workspace navigation so the relevant attention session opens
- expose the action in Settings → Shortcuts and translate its label
- carry the shortcut across the Electron browser-pane boundary

## QA

- `NODE_ENV=test npx vitest run packages/app/src/keyboard/keyboard-shortcuts.test.ts packages/app/src/keyboard/browser-shortcuts.test.ts packages/app/src/keyboard/route-shortcut.test.ts packages/app/src/stores/keyboard-shortcuts-store.test.ts packages/app/src/components/workspace-shortcut-targets-subscriber.test.tsx packages/app/src/i18n/resources.test.ts --bail=1` — 196 tests passed
- `npm run typecheck`
- `npm run lint`
- `npm run format:check`

Manual desktop/web QA was not run. Native mobile behavior is unaffected because global shortcuts are disabled there. No screenshots are included because this uses the existing data-driven shortcut settings row and does not introduce a new visual component or layout.

### CODE REVIEW

File: packages/app/src/keyboard/keyboard-shortcuts.ts [hunk 1]
```diff
@@ -538,6 +539,20 @@ const SHORTCUT_BINDINGS: readonly ShortcutBinding[] = [
...
+    action: "workspace.navigate.ready-waiting.next",
+    combo: "Alt+R",
+    when: { commandCenter: false },
+    help: {
+      id: "workspace-next-ready-waiting",
+      section: "navigation",
+      label: "Next ready/waiting workspace",
+      keys: ["alt", "R"],
+    },
+  },
```
[the remark this answers is not in the document]
Comment (colonelpanic8) [reply packages/app/src/keyboard/keyboard-shortcuts.ts:558]: Confirmed and fixed in cca2a3a70.

Verified the mechanism rather than assuming it: there is no global focus gate. `useKeyboardShortcuts` registers on `window` in **capture** phase (`use-keyboard-shortcuts.ts:333`) and calls `preventDefault()`/`stopPropagation()` on a match, so it beats xterm's `attachCustomKeyEventHandler`, which only runs later in the bubble phase on the helper textarea. Editable/terminal exclusion is opt-in per binding, in `matchesKeyboardShortcutContext` (`keyboard-shortcuts.ts:1180-1186`). With only `commandCenter: false`, Alt+R fired from the message input, any input/textarea/contentEditable, and a focused terminal.

The combo is also uniquely exposed compared to its neighbours: it is a bare letter with no mod key. On macOS Option+R emits `®`, and the binding matches by `event.code`, so the character substitution does not save it. In a terminal, readline binds Alt+R to `revert-line`.

Applied `when: { commandCenter: false, editable: false, terminal: false }`. I checked this against the surrounding conventions rather than pasting the suggestion: `editable: false` is what the Cmd+Shift+Arrow pane-focus bindings use (lines 613-649) to stay out of text selection, and `terminal: false` is what the non-mac Ctrl-based navigation bindings use. This binding needs both because it is reachable on every platform and in every scope.

Also added coverage: three no-match cases in `keyboard-shortcuts.test.ts` (macOS Option+R in the message input, Alt+R in a generic editable, Alt+R with a terminal focused), and updated the `KeyR` browser-shortcut prefix expectation, since `prefixFromCombo` now propagates `editable: false` into the Electron webview policy — so the guard carries across the browser boundary too.

File: packages/app/src/components/workspace-shortcut-targets-subscriber.test.tsx [hunk 2]
```diff
@@ -206,6 +207,63 @@ describe("WorkspaceShortcutTargetsSubscriber", () => {
...
+        <SidebarModelProvider>
+          <WorkspaceShortcutTargetsSubscriber enabled={true} />
+        </SidebarModelProvider>,
+      );
+    });
+
+    expect(useKeyboardShortcutsStore.getState().readyWaitingWorkspaceTargets).toEqual([
+      { serverId: "srv", workspaceId: "ws-waiting-new" },
+      { serverId: "srv", workspaceId: "ws-waiting-old" },
+      { serverId: "srv", workspaceId: "ws-ready" },
```
[the remark this answers is not in the document]
Comment (colonelpanic8) [reply packages/app/src/components/workspace-shortcut-targets-subscriber.test.tsx:266]: Declining this one — the objection is at odds with the file it is raised against, and the interaction it says is uncovered is in fact covered elsewhere.

`workspace-shortcut-targets-subscriber.test.tsx` is not new in this PR; it predates it (last touched in #1966). Every one of its five pre-existing assertions reads `useKeyboardShortcutsStore.getState().sidebarShortcutWorkspaceTargets` in exactly this way. The new case asserts `readyWaitingWorkspaceTargets` in the same shape, alongside the same component, for the same reason. Rewriting only the new one would make it the odd test out without making the file any less coupled to the store.

That coupling is also the actual contract here. This component renders `null` and its entire observable behaviour *is* what it publishes to the store — there is no user-facing surface to drive. The store shape is the seam, not an implementation detail leaking out.

On the coverage gap: the Option/Alt+R interaction is tested, just decomposed across the layer it belongs to rather than bundled into this file.

- `keyboard-shortcuts.test.ts` — keystroke to action: Alt+R and the macOS `®` Option-substitution both resolve to `workspace.navigate.ready-waiting.next`, plus (as of cca2a3a70) three no-match cases proving it stays out of editables and terminals.
- `route-shortcut.test.ts` — action to navigation: four cases covering advance, wrap-around, entry from a non-actionable workspace, and the empty-target no-op, each asserting a concrete `navigate-workspace` result.
- this file — the store targets those routing tests consume.

An end-to-end test would have to stand up the sidebar model, the store, the shortcut dispatcher, and the router together to re-assert what those three already pin down independently, and it would be the only test of its kind in this area. The seams are covered on both sides; I would rather keep the layering than add a slow test that duplicates it.

**Comment by boudra** [comment]
Closing this feature PR. Please discuss the workflow and shared need in [Discussions](https://github.com/getpaseo/paseo/discussions), following [CONTRIBUTING.md](https://github.com/getpaseo/paseo/blob/main/CONTRIBUTING.md).
